### PR TITLE
Enable host port

### PR DESCRIPTION
**Pull request by saiyan86**
**What this PR does / why we need it**:

Update Azure cni config file to enable hostport functionality of Kubernetes.

This PR contains changes that download portmap plugin from cni repo which will handle functionality of hostport specified in pod yaml

### CODE REVIEW

File: parts/k8s/kubernetesmastercustomscript.sh [hunk 1]
```diff
@@ -238,13 +238,13 @@ function configAzureNetworkPolicy() {
...
+    downloadUrl ${CNI_PLUGINS_URL} | tar -xz -C $CNI_BIN_DIR ./loopback ./portmap
     chown -R root:root $CNI_BIN_DIR
     chmod -R 755 $CNI_BIN_DIR
 
-    # Copy config file
-    mv $CNI_BIN_DIR/10-azure.conf $CNI_CONFIG_DIR/
-    chmod 600 $CNI_CONFIG_DIR/10-azure.conf
+    # Copy config file. After azure cni is released, we should change the 10-azure.conf to 10-azure.conflist
+    mv $CNI_BIN_DIR/10-azure.conflist $CNI_CONFIG_DIR/
+    chmod 600 $CNI_CONFIG_DIR/10-azure.conflist
```
Comment (sharmasushant) [inline parts/k8s/kubernetesmastercustomscript.sh]: This is dependent on Azure CNI drop that will have 10-azure.conflist file. We need to add the changes to pick up new drop of Azure CNI that has this file.

Comment (sharmasushant) [reply parts/k8s/kubernetesmastercustomscript.sh]: CNI version 1.0.1 is now available. please update this PR to pick 1.0.1 from https://github.com/Azure/azure-container-networking/releases/tag/v1.0.1 

This change requires v1.0.1 and v1.0.1 requires this change.

@saiyan86 @jackfrancis

Comment (jackfrancis) [reply parts/k8s/kubernetesmastercustomscript.sh]: See #2064, thanks for the notification @sharmasushant!

**Comment by jackfrancis** [comment]
See #2064

**Pull request closed without merging**